### PR TITLE
Opm widget

### PR DESCRIPTION
**Pull request by thasnim-cm**

### CODE REVIEW

File: src/components/organisms/OpmWidget.tsx [hunk 1]
```diff
@@ -1,71 +1,91 @@
...
+        description: w.description,
+      }));
+    const _channel = OPMNames.widgetDatas
+      .filter((item) => item.category === "Channel")
+      .map((w) => ({
+        data: w.property,
+        description: w.description,
+      }));
+    const _payment = OPMNames.widgetDatas
+      .filter((item) => item.category === "Payment")
```
Comment (AdithyaShaj) [inline src/components/organisms/OpmWidget.tsx]: Keep above constants in constants file

File: src/components/organisms/OpmWidget.tsx [hunk 2]
```diff
@@ -97,6 +119,13 @@ const OpmWidget = () => {
         data={payment}
         dataClassName="text-xs"
       />
+      {openOverlay && data && (
+        <CustomOverlayFocusRoom
+          ref={op}
+          header={data.description}
+          buttonContent="View Details"
```
Comment (AdithyaShaj) [inline src/components/organisms/OpmWidget.tsx]: Keep "View Details" in constants file

File: src/components/organisms/OpmWidget.tsx [hunk 3]
```diff
@@ -1,102 +1,131 @@
...
         columns={3}
         data={channel}
         dataClassName="text-xs"
       />
-      <GridTable
+      <GridCards
         title={FOCUS_ROOM_TITLES.PAYMENT}
         columns={3}
         data={payment}
         dataClassName="text-xs"
```
Comment (Kajish) [inline src/components/organisms/OpmWidget.tsx:122]: i can see a onclick functionality for locale and for shipment but not there for channel and payment is it missed out or is there any other reason

File: src/components/organisms/OpmWidget.tsx (hunk 1 above)
Comment (thasnim-cm) [reply src/components/organisms/OpmWidget.tsx]: done

File: src/components/organisms/OpmWidget.tsx (hunk 2 above)
Comment (thasnim-cm) [reply src/components/organisms/OpmWidget.tsx]: done

File: src/components/organisms/OpmWidget.tsx (hunk 3 above)
Comment (thasnim-cm) [reply src/components/organisms/OpmWidget.tsx:122]: It was missed. Added that

**Review by Kajish — APPROVED** [review]

**Review by AdithyaShaj — APPROVED** [review]

**Pull request merged**